pypeit/scripts/show_2dspec.py

```diff
@@ -115,8 +115,8 @@ def main(args):
     # Get waveimg
     mdir = head0['PYPMFDIR']+'/'
```

Comment:
Can we get away without adding the '/' here?  As much as possible, we should be using `os.path` to do this kind of stuff for us, but I realize there some lingering uses like this that we're still trying to nuke.